models/sipnet/inst/register.SIPNET.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<SIPNET>
```

Comment:
Perhaps this was because Ameriflux is an example where the met name in the settings file is not the same as the name in the database. But the met name in the settings file determines the file name of the registration file that we're looking for. 
```
<?xml version="1.0"?>
<Ameriflux>
  <scale>site</scale>
  <format>
    <id>38</id>
    <name>AmeriFlux.level2.h.nc</name>
    <mimetype>application/x-netcdf</mimetype>
    <inputtype>ameriflux.zip</inputtype>
  </format>
</Ameriflux>
```